packaging/install/drain-cleaner/kubernetes/webhook-certificates/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Generate CA
+cfssl genkey -initca ca.json | cfssljson -bare ca
+
+# Sign webhook certificate
+cfssl genkey webhook.json | cfssljson -bare webhook
+cfssl sign -config config.json -profile server -ca ca.pem -ca-key ca-key.pem webhook.csr webhook.json | cfssljson -bare webhook
+
+# Create CRT bundles
+cat webhook.pem > webhook-bundle.crt
+cat ca.pem >> webhook-bundle.crt
+
+# Convert keys to PKCS8
+openssl pkcs8 -topk8 -nocrypt -in ca-key.pem -out ca.key
```

Comment:
Do we have to depend on _both_ cfssl and openssl?

Reply:
It is just an example script ... at the end everyone can generate the certificates anyway they want. They do not have to use these tools.